client/src/types/UiContextType.d.ts

```diff
@@ -0,0 +1,15 @@
+type UiContextType = {
+    isHamburgerOpen: boolean,
+    toggleHamburger: () => void,
+    closeHamburger: () => void,
+
+    isSuperuser: boolean,
+    isFinanceAdmin: boolean,
+    isEventAdmin: boolean,
+    isYogaAdmin: boolean,
+    setAccess: (a: boolean, b: boolean, c: boolean, d: boolean) => void,
```

Comment:
Itt nem mukodik valami olyan, hogy `(...a: boolean[]) => void` ahelyett, hogy felsoroljuk a parametereket?

Reply:
I am not sure!
In TS parameter names are just placeholders. You can find in our code that I use parameter names like 'a', 'b' in type definitions. But in real functions they are 'firstName' and 'lastName'..
So the POSITION is important and not the names.